enaml/src/colorext.cpp

```diff
@@ -160,9 +160,30 @@ Color_getset[] = {
 };
 
 
+static PyObject*
+Color__reduce__( Color* self, void* context )
+{
+    PyObject* cls = PyObject_Type(reinterpret_cast<PyObject*>( self ));
+    if ( !cls )
+        return 0;
+    uint32_t a = ( self->argb >> 24 ) & 255;
```

Comment:
Can you use `0xFF` instead of `255`. I use hex notation when performing bit-masking, since it makes it a bit more clear what's going on.

Reply:
Agree. Updated.